index.html

```diff
@@ -110,6 +110,7 @@ <h2>Input</h2>
 					<option value="slantsplat">"/* ... */ " CSS </option>
 					<option value="xml">"&lt;!-- ... --&gt;" XML</option>
 					<option value="pipe">"|" Reddit pipe</option>
+          <option value="codeblock">"```" (triple backtick) Markdown codeblock (reddit, Github, Discord, Slack)</option>
```

Comment:
You used spaces for indentation but the surrounding indentation is made with tabs.

```suggestion
					<option value="codeblock">"```" (triple backtick) Markdown codeblock (reddit, Github, Discord, Slack)</option>
```